docs/source/models/intervention_models/neonatal/cpap_intervention.rst

```diff
@@ -87,7 +87,45 @@ We will then add an attribute to each simulant indicating whether the birth occu
 
 We will then use the conditional probabilities for simulants with and without access to determine which simulants die from RDS.
 
-A `2020 Cochrane review <https://pmc.ncbi.nlm.nih.gov/articles/PMC8094155/>`_ found a relative risk of 0.53 (95% CI 0.34-0.83) of RDS mortality for neonates with access to CPAP.   (Note that the population that this effect size applies to is preterm infants with "respiratory failure becoming evident soon after birth".)
+A `2020 Cochrane review <https://pmc.ncbi.nlm.nih.gov/articles/PMC8094155/>`_ found a relative risk of 0.53 (95% CI 0.34-0.83) of overall mortality for neonates with access to CPAP.   (Note that the population that this effect size applies to is preterm infants with "respiratory failure becoming evident soon after birth".)
+
+So specifically, the preterm with RDS cause-specific mortality risk for an individual simulant, :math:`i`, as derived from the :ref:`neonatal preterm birth cause model document <2021_cause_preterm_birth_mncnh>` (:math:`\text{CSMRisk}^{\text{preterm with RDS}}_{\text{BW},\text{GA}}`) should be further modified by CPAP intervention access as follows:
+
+.. math::
+
+  \text{CSMRisk}^{\text{preterm with RDS}}_{\text{BW},\text{GA}, i} = \text{CSMRisk}^{\text{preterm with RDS}}_{\text{BW},\text{GA}} * (1 - \text{PAF}) * \text{RR}_i
+
+Where,
+
+.. list-table:: CPAP intervention parameters
+  :header-rows: 1
+
+  * - Parameter
+    - Definition
+    - Value
+    - Note
+  * - :math:`\text{CSMRisk}^{\text{preterm with RDS}}_{\text{BW},\text{GA}, i}`
+    - Mortality risk due to preterm with RDS for a given simulant :math:`i` with a specific birth weight and gestational age exposure following modification from the CPAP intervention
+    - See equation above
+    - 
+  * - :math:`\text{CSMRisk}^{\text{preterm with RDS}}_{\text{BW},\text{GA}}`
+    - Mortality risk due to preterm with RDS for a simulant with a given birth weight and gestational age exposure before modification from the CPAP intervention
+    - Derived from instruction on the :ref:`neonatal preterm birth cause model document <2021_cause_preterm_birth_mncnh>`
+    - 
```

Comment:
It could be confusing that the notation here is slightly different from what's on the preterm birth cause page, which I think uses $\text{CSMRisk}^{\text{preterm with RDS}}\_i$ and $\text{CSMRisk}^{\text{preterm with RDS}}\_{\text{BW},\text{GA}}$ for these, respectively.